Demo/EarlGreyContribs/EarlGreyContribs.xcodeproj/project.pbxproj

```diff
@@ -105,6 +105,7 @@
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
+		FD8277DB1D677D0D00C7C2AD /* EarlGrey.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; name = EarlGrey.swift; path = "../../../gem/lib/earlgrey/files/Swift-3.0/EarlGrey.swift"; sourceTree = "<group>"; };
```

Comment:
The Contribs project is specifically created to use EarlGrey from HEAD and not test the Pods flow out. Adding Pods would quite complicate this flow up.